storage_service/locations/models/space.py

```diff
@@ -587,23 +591,29 @@ def browse_rsync(self, path, ssh_key=None):
 
         :param path: Path to query, including user & hostname. E.g. user@host:/path/to/browse/  Must end in with / to browse directories.
         :param ssh_key: Path to the SSH key on disk. If None, will use default.
+        :param bool assume_rsync_daemon: If true, will use rsync daemon-style commands instead of the default rsync with remote shell transport
+        :param rsync_password: used if assume_rsync_daemon is true, to specify value of RSYNC_PASSWORD environment variable
         :return: See docstring for Space.browse
         """
         if ssh_key is None:
             ssh_key = '/var/lib/archivematica/.ssh/id_rsa'
 
-        # Get entries
-        command = [
-            'rsync',
-            '--protect-args',
-            '--list-only',
-            '--exclude', '.*',  # Ignore hidden files
-            '--rsh', 'ssh -i ' + ssh_key,  # Specify identify file
-            path]
+        # Form command string used to get entries
+        command = ['rsync',
+                   '--protect-args',
+                   '--list-only',
+                   '--exclude', '.*',  # Ignore hidden files
+                   ]
+        if not(assume_rsync_daemon):
```

Comment:
`not` is a keyword, not a function.
I think it's better to type it as: `if not assume_rsync_daemon:`